src/libraries/System.Runtime.InteropServices/tests/TestAssets/SharedTypes/NonBlittable.cs

```diff
@@ -477,7 +502,7 @@ public void FromManaged(List<T> managed, Span<TUnmanagedElement> buffer)
 
                 _list = managed;
                 // Always allocate at least one byte when the list is zero-length.
-                int spaceToAllocate = Math.Max(managed.Count * sizeof(TUnmanagedElement), 1);
+                int spaceToAllocate = Math.Max(managed.Count, 1);
```

Comment:
I don't understand this. How is the element count of a `List<T>` translate into a byte unit?

Reply:
We don't need to translate this to bytes because our Span type is `Span<TUnmanagedElement>`.

Reply:
But that doesn't entirely follow. We use `spaceToAllocate` on line 512 and interpret it as byte units passed to `Marshal.AllocCoTaskMem`. Perhaps we need to update that instead?

Reply:
Also, we should also update the name of that variable as it implies "bytes" when it should be "element units".